src/clients/Deluge.ts

```diff
@@ -66,19 +81,27 @@ export default class Deluge implements TorrentClient {
 		} catch (networkError) {
 			throw new CrossSeedError(networkError);
 		}
-		const connectedResponse = await this.call("web.connected", [], 0);
+		const connectedResponse = await this.call<boolean>(
+			"web.connected",
+			[],
+			0
+		);
 
 		if (!connectedResponse.result) {
 			logger.warn(
 				"Deluge WebUI disconnected from daemon...attempting to reconnect."
 			);
-			const webuiHostList = await this.call("web.get_hosts", [], 0);
-			const connectResponse = await this.call(
+			const webuiHostList = await this.call<object>(
+				"web.get_hosts",
+				[],
+				0
+			);
+			const connectResponse = await this.call<undefined>(
```

Comment:
can we create a higher-fidelity type than `object` for webuiHostList? Based on line 101 where we use it like `webuiHostList.result[0][0]` it seems like it's an array of arrays of something?